supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc

```diff
@@ -3,7 +3,7 @@
 ==== image:images/yes.png[yes] Emacs (noun)
 *Description*: "Emacs" is a text editor available in Unix. Like vi, Emacs is a screen editor. Unlike vi, Emacs is not an insertion mode editor, meaning that any character typed in Emacs is automatically inserted into the file, unless it includes a command prefix.
 
-If referring to the program, use "Emacs," for example, "Source-Navigator supports Emacs or vi commands." If referring to the shell prompt command, use `emacs`. The command should always be properly marked up, for example, "At the prompt, type `emacs`." The complete and correct name is "GNU Emacs."
+If referring to the program, use "Emacs," for example, "Source-Navigator supports Emacs or vi commands." If referring to the shell prompt command, use `emacs`. Always mark up the command correctly, for example, "At the prompt, type `emacs`." The complete and correct name is "GNU Emacs."
```

Comment:
@ndeevy PR looks good. Bringing up one thing only because I see above that you changed "correct." to "correct". and another instance of double quotation with punctuation. Your changes there are correct, but for a separate PR, we should ensure consistency with double quotes with other punctuation, like "word." vs. "word".  per [IBM guidance](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-quotation-marks#quotation-marks-with-other-punctuation). The ruling is basically always "word". "word", etc. except when the quoted portion is a complete sentence on its own with its own punctuation, or if the component has its own punctuation.

This sample here is a perfect example of both cases: Everything in this line is exactly right, with cases of internal and external periods used perfectly, except the last instance, which should be "GNU Emacs".

Nothing you've done here of course, just pointing out the need for some alignment throughout based on the corrections you made previously. Something to consider.

Reply:
Definitely something to consider, TY @sterobin  I've found times where I was unsure about this, which means others might too so it'd be good to have guidance.